README.md

```diff
@@ -1,15 +1,245 @@
+  - [Benchmarking](#benchmarking)
+  - [Workflow summaries](#workflow-summaries)
+      - [Metadata](#metadata)
+      - [Component tools](#component-tools)
+  - [Additional notes](#additional-notes)
+  - [Help/FAQ/Troubleshooting](#helpfaqtroubleshooting)
+  - [Acknowledgements/citations/credits](#acknowledgementscitationscredits)
+
+## Description
+Somatic-shortV-nf is a Nextflow pipeline for identifying somatic short variant events in Illumina short read whole genome sequence data. 
+We have followed the [GATK Best practices](https://gatk.broadinstitute.org/hc/en-us/articles/360035894731-Somatic-short-variant-discovery-SNVs-Indels-) .  
+
+There are three main steps in the process of calling Somatic Short Variants:
+
+(1) **Creation of Somatic short variants Panel of Normals (PoN)** : 
+<br>This involves converting the Normal BAMs to PON.   
+The PON's are -
+  * Made from normal samples i.e. the samples derived from healthy tissue and 
+  * Their main purpose is to capture recurrent technical artifacts in order to improve the results of the variant calling analysis.
+  * Please use the github repository [SomaticShortV_makePON-nf](https://github.com/Sydney-Informatics-Hub/Somatic-shortV-makePON-nf) to create the PoN. 
+<br>
+
+(2) **Call the somatic short variants for a Tumor-Normal pair** :
+<br>The two main steps involved in calling  Somatic Short Variants are - 
+  * Generate a large set of candidate somatic variants.
+  * Filter the candidate somatic variants into a more confident set of somatic variant calls.
+
+- Calling candidate somatic short variants involves the use of the tool [Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/360035531132) which calls both SNVs and indels simultaneously by generating a local assembly of haplotypes in an active region, de-novo. When Mutect2 sees a region with somatic variations, it dis-regards the existing mapping information completely and re-assembles the reads in that region in order to generate candidate variant haplotypes.
+
+* This is followed by calculating contamination using the tools [GetPileupSummaries](https://gatk.broadinstitute.org/hc/en-us/articles/9570416554907-GetPileupSummaries) and [CalculateContamination](https://gatk.broadinstitute.org/hc/en-us/articles/9570322332315-CalculateContamination)
+* The next step is to learn the parameters of a model for orientation bias using the tool [LearnReadOrientationModel](https://gatk.broadinstitute.org/hc/en-us/articles/360051305331-LearnReadOrientationModel)
+* The [FilterMutectCalls](https://gatk.broadinstitute.org/hc/en-us/articles/360036856831-FilterMutectCalls) tool then applies filters to the raw output of Mutect2. 
+
+
+(3) **Annotating the variants** 
+<br> An external genomic variant annotations and functional effect prediction tool [SnpEff](http://pcingola.github.io/SnpEff/) is used for annotating the filtered variants (such as amino-acid changes etc). Please refer to the above link for SnpEff details.
+
```

Comment:
The formatting you've used here isn't easy to read and duplicates what is already described in the GATK best practice documentation you've linked. I suggest removing all this text or at least significantly reducing to a couple of sentences.